src/Infrastructure/Controller/Regulation/DeleteRegulationController.php

```diff
@@ -59,20 +58,25 @@ public function __invoke(Request $request, string $uuid): Response
             // Don't fail, as DELETE is an idempotent method (see RFC 9110, 9.2.2).
             return new RedirectResponse(
                 url: $this->router->generate('app_regulations_list', [
-                    'tab' => RegulationOrderRecordStatusEnum::DRAFT,
+                    'tab' => 'temporary',
                 ]),
                 status: Response::HTTP_SEE_OTHER,
             );
         }
 
+        // Get before the entity gets deleted.
+        $endDate = $regulationOrderRecord->getRegulationOrder()->getEndDate();
```

Comment:
Ça m'a un peu surpris ici. Il semblerait qu'on ait potentiellement des requêtes SQL cachées liées au fait qu'on récupère toute l'entité `regulationOrderRecord` dans `GetRegulationOrderRecordByUuidQuery`.

En effet, si je mettais cette ligne après l'exécution de la `DeleteRegulationCommand`, Doctrine lançait une exception du type "RegulationOrder with uuid = ... does not exist", me laissant penser qu'il était en train de faire une requête SQL pour récupérer la `endDate`.